apps/src/dance/ai/AiBlockPreview.tsx

```diff
@@ -16,40 +16,35 @@ const AiBlockPreview: React.FunctionComponent<AiBlockPreviewProps> = ({
 }) => {
   const blockPreviewContainerRef = useRef<HTMLSpanElement>(null);
   const workspaceRef = useRef<Workspace | null>(null);
+  const isRtl = useSelector((state: {isRtl: boolean}) => state.isRtl);
 
   // Create the workspace once the container has been rendered.
   useEffect(() => {
     if (!blockPreviewContainerRef.current) {
       return;
     }
 
-    const emptyBlockXml = Blockly.utils.xml.textToDom('<xml></xml>');
+    const xml = `
```

Comment:
Can we create a separate util for generating this XML?

As a side note, seeing this XML sent me down a bit of a rabbit hole in terms of how we're creating and loading blocks, and if maybe we should be using this method more (rather than creating blocks directly in a workspace). We could even potentially use JSON here instead. Added a note to the tech debt doc to circle back on.

Reply:
Good call; added a separate util function.

It was [this](https://groups.google.com/g/blockly/c/7MB1THSMFiY/m/XYWvQ4DuBAAJ) post that gave the hint that creating blocks directly wasn't getting us the proper layout.  It makes sense that we might then unify around that approach.

Also agreed that JSON might be better than XML here, since we're generally trying to move towards JSON for Blockly.